microsetta_interface/tests/test_integration.py

```diff
@@ -457,14 +457,13 @@ def test_only_untaken_secondarys_available(self):
         self.assertPageTitle(resp, 'Account Samples')
         data = self._html_page(resp)
 
-        # we've taken the fermented food survey, so we should not
-        # observe its URL in the rendered page
-        # TODO: this check will likely break if/when survey editing is allowed
+        # Now that surveys can be re-taken, we should observe the URL for
+        # the fermented food surveym even though we've already taken it.
```

Comment:
Will there be an indication on the survey page letting them know that they have already taken the survey (and when)?

Reply:
There will be a visual indicator on the My Profile page reflecting how much of a given survey they've completed. Since we're moving towards the idea of "Building a Profile," we're not displaying the timestamp they last took it.

As with the warning, this will be wrapped into my PR.

Reply:
Thanks!